contrib/debian/rules

```diff
@@ -37,6 +37,21 @@ HAVE_XENSTAT = 1
 XENSTAT_CONFIG = -DENABLE_PLUGIN_XENSTAT=On
 endif
 
+ifeq ($(ENABLE_SENTRY),true)
+    BUILDING_ENVIRONMENT ?= Unknown
+    VERSION ?= Unknown
+    BUILD_DESTINATION ?= Unknown
+    SENTRY_CONFIG := -DENABLE_SENTRY=On \
+        -DNETDATA_SENTRY_ENVIRONMENT=$(BUILDING_ENVIRONMENT) \
+        -DNETDATA_SENTRY_RELEASE=$(VERSION) \
+        -DNETDATA_SENTRY_DIST=$(BUILD_DESTINATION) \
+        -DNETDATA_SENTRY_DSN=$(SENTRY_DSN) \
+        -DCMAKE_BUILD_TYPE=Debug
```

Comment:
Are we sure we want a straight Debug build and not RelWithDebInfo?

Also, independent of this we _probably_ want RelWithDebInfo for non-sentry builds so that we get proper debug symbol packages.